src/app/login/login.component.html

```diff
@@ -0,0 +1,25 @@
+<form [formGroup]="loginForm" (ngSubmit)="onSubmit()">
+    <div class="wrap-entrance">
+      <span class="form-title">
+        Entrance
+      </span>
+      <div class="wrap-input">
+        <input id="name" class="input" type="text" formControlName="name" placeholder="Login" [ngClass]="{ 'is-invalid': submitted && f.name.errors }">
```

Comment:
Instead of `[ngClass]="{ 'is-invalid': submitted && f.name.errors }"` we can use `[class.isInvalid]="submitted && f.name.errors"`. I think it looks simpler.